### PR TITLE
Ruby/Sinatra/DynamoDB example

### PR DESCRIPTION
**Pull request by icole**
This is inspired from @pgrzesik's great aws-python-flask-api example that I ported to Ruby/Sinatra. I am still learning serverless, so welcome any feedback or suggestions on if this could have been implemented any better.

### CODE REVIEW

File: aws-ruby-sinatra-dynamodb-api/serverless.template.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: aws-ruby-sinatra-dynamodb-api
```
Comment (pgrzesik) [inline aws-ruby-sinatra-dynamodb-api/serverless.template.yml:1]: I think we can remove this file, as this was additionally needed to publish an example as a template for https://app.serverless.com/

**Review by pgrzesik — CHANGES_REQUESTED** [review]
Thank you @icole :bow: I'm sorry to not get to this sooner, but I've somehow missed the notification and noticed it only now. I've quickly tested it and it works great :+1: Let's just remove the `serverless.template.yml` and we're good to go

**Comment by icole** [comment]
@pgrzesik Apologies for my delay now! I as well did not seem to get a notification. I went ahead and removed that file. Please let me know if there is anything else I can do.

**Review by pgrzesik — APPROVED** [review]
Thank you @icole - awesome job :clap:

**Pull request merged**